web/src/libs/express-utilities.js

```
@@ -0,0 +1,15 @@
+/**
+ * Normalize a port into a number, string, or false.
+ */
+export const normalizePort = (val) => {
+  var port = parseInt(val, 10)
+  if (isNaN(port)) {
+    // named pipe
```

I'm guessing this is copied code from somewhere? The web server is only going to be handling http connections, not named pipes, so this should throw an error to let us know the requested PORT is not a number to make config validation/debug easier